[2024-06-12T00:13:26.304+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_etl.get_dir scheduled__2024-01-08T12:00:00+00:00 [queued]>
[2024-06-12T00:13:26.309+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_etl.get_dir scheduled__2024-01-08T12:00:00+00:00 [queued]>
[2024-06-12T00:13:26.310+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2024-06-12T00:13:26.323+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_dir> on 2024-01-08 12:00:00+00:00
[2024-06-12T00:13:26.326+0000] {standard_task_runner.py:57} INFO - Started process 403 to run task
[2024-06-12T00:13:26.328+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_data_etl', 'get_dir', 'scheduled__2024-01-08T12:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/data_etl_dag.py', '--cfg-path', '/tmp/tmp9b0kdjzy']
[2024-06-12T00:13:26.329+0000] {standard_task_runner.py:85} INFO - Job 40: Subtask get_dir
[2024-06-12T00:13:26.359+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_data_etl.get_dir scheduled__2024-01-08T12:00:00+00:00 [running]> on host c09d00e80527
[2024-06-12T00:13:26.411+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Solarev Ruslan' AIRFLOW_CTX_DAG_ID='dag_data_etl' AIRFLOW_CTX_TASK_ID='get_dir' AIRFLOW_CTX_EXECUTION_DATE='2024-01-08T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-08T12:00:00+00:00'
[2024-06-12T00:13:26.412+0000] {logging_mixin.py:151} INFO - ['dags', 'logs', '***.cfg', 'plugins', 'data', 'tasks', 'config']
[2024-06-12T00:13:26.413+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-06-12T00:13:26.418+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_data_etl, task_id=get_dir, execution_date=20240108T120000, start_date=20240612T001326, end_date=20240612T001326
[2024-06-12T00:13:26.460+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-06-12T00:13:26.470+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
